src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        ArrayList<Task> taskList = new ArrayList<Task>();
+        String taskName = "";
+        Scanner sc = new Scanner(System.in);
+        System.out.println("\t____________________________________________________________\r\n"
```

Comment:
Maybe you can consider storing the ____ String as a class-level constant variable since it's used multiple times and a variable name would be more readable